### PR TITLE
Retries and retry_interval support to ensure robustness in case of temporary network outtage

### PR DESCRIPTION
**Pull request by matias-easymile**

### CODE REVIEW

File: pytest_loki/__init__.py [hunk 1]
```diff
@@ -178,15 +192,18 @@ def _push_to_loki(self, streams: List[Tuple[Dict, List[str]]]) -> str:
...
-        else:
-            if not r.ok:
-                error_msg = f"Failed to send tests resulsts to loki: {r.status_code}: {r.content}"
-        return error_msg
+        for i in range(self._retry_attempts):
+            try:
+                r = requests.post(self._loki_url, json=payload, auth=self._basic_auth)
+                r.raise_for_status()  # This will raise an HTTPError if the request returned a status error
+                return  # If no exception was raised, the request was successful, return from the function
+            except requests.RequestException as e:
```
Comment (Patazerty) [inline pytest_loki/__init__.py]: Should we switch to a catch-all try ? We've seen that requests can end up raising an urllib exception that does not fall under the `requests.RequestException` class

File: README.md [hunk 2]
```diff
@@ -18,26 +18,45 @@ If no loki url is provided the plugin will not do anything.
 
 ## Usage
 
+When invoking `py.test`, provide the following arguments:
+## Usage
+
```
Comment (Patazerty) [inline README.md]: This looks like a duplicate

**Review by Patazerty — CHANGES_REQUESTED** [review]
Thanks for your PR ! Except the small typo in the doc and maybe the tweak in the exception catching, LGTM

File: README.md (hunk 2 above)
Comment (matias-easymile) [reply README.md]: fixed

**Pull request merged**